### PR TITLE
165 day of week app

### PR DESCRIPTION
**Pull request by sonamgupta136**
## Week Day Predictor App

    A simple week  day predictor app using HTML , CSS and JAVASCRIPT.

## Use Of Project :

    This app displays the current day of the week with amazing quotes. 

## Technologies Used:
     - HTML
     - CSS
     - JAVASCRIPT

## Steps to Use:

---

 - Download or clone the repository

```
    "git clone https://github.com/Ayushparikh-code/Web-dev-mini-projects.git"
```

 - Go to the directory
 - Run the index.html file
 - Result is here!!!

## Screenshot:
Screenshot (18)

### CODE REVIEW

**Review by Ayushparikh-code — CHANGES_REQUESTED** [review]
hey @sonamgupta136 u added two same readme.md so delete any of the one , 
and do update and add your project name & description in Index.md file(https://github.com/Ayushparikh-code/Web-dev-mini-projects/blob/main/Index.md)

**Review by chicken-biryani — COMMENTED** [review]
@sonamgupta136 You've done really very well.

Can u just add a screenshot above the Demo and it's good to go! Excellent!

**Comment by sonamgupta136** [comment]
@chicken-biryani  #165 please review now.

File: Index.md [hunk 1]
```diff
@@ -1,63 +1,59 @@
...
-| [Form Validation](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/form-validation)                         |A basic sign up page with all the validations in javascript.   |     
-| [Age Calculator](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/age-calculator)                           | A website which can be used to calculate age of a person in days,months and years . |
-| [BMI Calculator](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/bmi-calculator)                           | A basic website which can be used to calculate body mass index |
-| [Glassmorphism Animation Form](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/Glassmorphism-Form)      |A basic login page with Glassmorphism Animations |       
-| [Battery Percentage Indicator](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/Battery%20Indicator)      |This is a battery indicator app which is used to know battery percentage of the system. |
```
Comment (harshita2216) [inline Index.md:57]: Read my review in #189

**Review by Ayushparikh-code — APPROVED** [review]

**Pull request merged**